Dockerfile

```diff
@@ -46,6 +46,7 @@ RUN dpkg --add-architecture i386 \
         jq \
         python3-minimal \
         python3-pkg-resources \
+	ssh \
```

Comment:
indentation is off

Reply:
sorry, what do you mean by this?

Reply:
Like the `ssh` is under-indented. As in all other packages are indented with 8 spaces but the line you added is indented with 6 spaces.